.github/workflows/bdd.yaml

```diff
@@ -0,0 +1,28 @@
+name: BDD tests
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  bdd:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go-version:
+          - "1.17"
```

Comment:
Is this the preferred go version for bdd tests? Why not any more recent version?